deepchecks/vision/utils/image_functions.py

```diff
@@ -189,7 +189,7 @@ def prepare_thumbnail(
         image = ensure_image(image, copy=False)
 
     img_bytes = io.BytesIO()
-    image.save(fp=img_bytes, format='PNG')
+    image.save(img_bytes, optimize=True, quality=60, format='jpeg')
```

Comment:
This is on purpose?

Reply:
this change should be in another pr probably